test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/GroupCollectionRequest.java

```diff
@@ -0,0 +1,150 @@
+// ------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+package com.microsoft.graph.requests.extensions;
+
+import com.microsoft.graph.http.IRequestBuilder;
+import com.microsoft.graph.core.ClientException;
+import com.microsoft.graph.concurrency.ICallback;
+import com.microsoft.graph.models.extensions.Group;
+import java.util.Arrays;
+import java.util.EnumSet;
+import javax.annotation.Nullable;
+import javax.annotation.Nonnull;
+
+import com.microsoft.graph.options.QueryOption;
+import com.microsoft.graph.core.IBaseClient;
+import com.microsoft.graph.http.BaseCollectionRequest;
+import com.microsoft.graph.concurrency.IExecutors;
+import com.microsoft.graph.requests.extensions.GroupCollectionResponse;
+import com.microsoft.graph.requests.extensions.GroupCollectionRequestBuilder;
+import com.microsoft.graph.requests.extensions.GroupCollectionRequest;
+
+// **NOTE** This file was generated by a tool and any changes will be overwritten.
+
+/**
+ * The class for the Group Collection Request.
+ */
+public class GroupCollectionRequest extends BaseCollectionRequest<Group, GroupCollectionResponse, GroupCollectionPage> {
```

Comment:
Is $count query option not applicable for  this class?

Reply:
java currently doesn't have any method being generated for count...which is amazing to me that nobody complained about it so far!
it's captured here and will be tackled in a subsequent PR https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/issues/318